src/js/components/FilterByTaskState.js

```diff
@@ -56,13 +57,22 @@ class FilterByTaskState extends React.Component {
   }
 
   render() {
+    let {inverseStyle} = this.props;
+
+    let dropdownMenuClassSet = classNames({
```

Comment:
makes sense, I'll make a new PR for that since it touches a lot more than just NodeSidePanel